README.md

```diff
@@ -1731,6 +1730,17 @@ Disabling Auto-Init means that you _must_ explicitly call `terragrunt init` prio
 
 If Auto-Init is disabled, and terragrunt detects that `terraform init` needs to be called, then terragrunt will fail.
 
+### Auto-Retry
+
+_Auto-Retry_ is a feature of `terragrunt` that will automatically address situations where a `terraform` command needs to be re-run.
+
+Terraform can fail with transient errors which can be addressed by simply retrying the command again. In the event `terragrunt` finds one of these errors, the command will be re-run again automatically.
```

Comment:
How is the list of transient errors maintained? I presume somewhere in the code we have a default list; perhaps we can link to the file (e.g., `retryable-errors.go`)? Also, can the user specify extra ones in the Terragrunt config?

Reply:
Also, consider adding an example here: "You run `terragrunt apply` and get an error such as [...] Terragrunt knows this is a transient error and will automatically retry..."